lib/rules/report-message-format.js

```diff
@@ -70,7 +70,7 @@ module.exports = {
         messagesObject.value.properties
           .filter(prop => prop.type === 'Property')
           .map(prop => prop.value)
-          .forEach(processMessageNode);
+          .forEach(propValue => processMessageNode(propValue));
```

Comment:
maybe just my preference, I like the unchanged code, for more concise. :)

Reply:
Sounds good, I disabled unicorn/no-array-callback-reference](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-callback-reference.md) for now and removed the autofixes.